js/controllers/todoCtrl.js

```diff
@@ -29,28 +29,41 @@ function ($scope, $location, $firebaseArray, $sce, $localStorage, $window) {
 
 
 var splits = $location.path().trim().split("/");
-var roomId = angular.lowercase(splits[1]);
-if (!roomId || roomId.length === 0) {
-	roomId = "all";
-}
+// Restructure the room as an object
+var room = {
+    roomid: angular.lowercase(splits[1]),
+};
+//var roomId = angular.lowercase(splits[1]);
+if (!room.roomid || room.roomid.length === 0) {
+	room.roomid = "all";
+};
 
 // Kaichen's firebase account
 var firebaseURL = "https://cmkquestionsdb.firebaseio.com/";
+// Backup DB (TEST ONLY)
+//var firebaseURL = "https://questionstestdb.firebaseio.com/";
 
 // create variables for firebase DB
-$scope.roomId = roomId;
-var url = firebaseURL + roomId + "/questions/";
-var urlReplies = firebaseURL + roomId + "/replies/";
-var echoRef = new Firebase(url);
+
```

Comment:
It looks like you merged this part correctly. 
There were no issues while testing.